application.go

```diff
@@ -99,6 +98,8 @@ type Application struct {
 	LastTaskFailure       *LastTaskFailure        `json:"lastTaskFailure,omitempty"`
 	Fetch                 *[]Fetch                `json:"fetch,omitempty"`
 	IPAddressPerTask      *IPAddressPerTask       `json:"ipAddress,omitempty"`
+	Env                   map[string]string       `json:"env,omitempty"`
```

Comment:
Is it intentional that we drop the pointer? This was previously required so that we could clear the environment variables. (`encoding/json` drops empty values with the `omitempty` tag.)